lib/decompress/huf_decompress.c

```diff
@@ -67,6 +67,12 @@
 #define HUF_STATIC_ASSERT(c) { enum { HUF_static_assert = 1/(int)(!!(c)) }; }   /* use only *after* variable declarations */
 
 
+/* **************************************************************
+*  Byte alignment for workSpace management
+****************************************************************/
+#define ALIGN(x, a) ALIGN_MASK((x), (a) - 1)
```

Comment:
namespace : `ALIGN` name risks being too generic, it may collide with another macro using same name into a 3rd-party project.
Prefer `HUF_ALIGN` and `HUF_ALIGN_MASK`.